core/idl/wallet/algorand/idl.djinni

```diff
@@ -35,12 +35,12 @@ AlgorandAssetParams = record {
     # A URL where more information about the asset can be retrieved
     url: optional<string>;
     # Specifies whether holdings in this asset are frozen by default
-    defaultFrozen: optional<bool>;
+    defaultFrozen: bool;
 
     # The total number of units of this asset
     total: optional<string>;
     # The number of digits to use after the decimal point when displaying this asset
-    decimals: optional<i32>;
+    decimals: optional<string>;
```

Comment:
Instead of use a string which brings conversation mechanism and probably new unmanaged errors, I suggest to key the initial type which is `i32` and create placeholder value.
This placeholder can be used to skip the value if wanted

Reply:
We used `string` everywhere for numbers in algorand (because we couldn't use u64), so I think it wouldn't make a lot of sense to use a placeholder here

Reply:
Thinking twice, I think I agree with @barjonr: in all 3 cases (Algorand, Cosmos, Ethereum) most values are passed as optional<string>, so it would be more consistent to adapt the few ones that were passed as number?